kokoro/interop.sh

```diff
@@ -15,8 +15,14 @@
 
 set -ex
 
-# change to grpc repo root
+# change to grpc-dotnet repo root
 cd $(dirname $0)/..
 
-# TODO(jtattermusch): run interop tests
-echo "Interop tests not yet setup."
+git clone https://github.com/grpc/grpc ./../grpc
```

Comment:
Is this a pattern that's common in grpc infrastructure? I'm a little skeptical of cloning entire repos in parent directories. How about submodules if we are going that route or is there a directory structure requirement for this to work?

I'm not opposed to it but it does seem like an odd pattern.

Reply:
Yes, this is a pattern we are using to run other interop test suites as well. So it's basically a "standard" pattern.
Basically the testing script expects the directories with clones of language implementations to live side by side (one advantage is that makes local testing easier).

https://github.com/grpc/grpc/blob/a41a381302376488a779422e50a6c2d4bd7049ca/tools/internal_ci/helper_scripts/prepare_build_interop_rc#L27